### PR TITLE
Fix the localization in WCUND.1301

### LINKED ISSUE #695: [LOCALIZATION] | WCUND.1301
**Mod Version**
9288221b

**Please explain your issue in as much detail as possible:**
"I will not betray [Godname]" looks a bit weird in this context
How about change it into something like "Forgive me, [Godname]"?

**Upload screenshots of the problem localization:**

<details>

image

</details>

### PR DESCRIPTION
**Pull request by DJScias**
Hello there,

Self-explanatory.

This PR should fix #695. 

Third option in screenshot:
image

### CODE REVIEW

**Comment by zumbak04** [comment]
> I'd say it's almost perfect... Can we change "the light" to "light" in the line where it's defined?

No, other lines require Light with article.

**Comment by arithon** [comment]
Then how about changing it to "I hope <the light> will forgive me." ?

**Comment by zumbak04** [comment]
> Then how about changing it to "I hope will forgive me." ?

Works... I think, @DJScias , whachu think?

**Comment by DJScias** [comment]
I guess you mean it in the following way? If so I can do that:

I hope **the Light** will forgive me.

**Comment by zumbak04** [comment]
> I guess you mean it in the following way? If so I can do that:
> 
> I hope **the Light** will forgive me.

Yep, sounds fine to me.

**Comment by DJScias** [comment]
Requested changes made, spoke to @arithon on Discord to get the correct German translation as well.

**Comment by arithon** [comment]
I got no idea, why "the light" disappeared from my previous comment... good thing we got discord :P

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -2655,7 +2655,7 @@ EVTDESC_WCUND_1301;Your past gods turned their back on you after you were strick
 EVTOPTA_WCUND_1301;I shall bear this burden, but I will not forsake my freedom.;;Ich werde diese Last tragen, aber ich werde meine Freiheit nicht aufgeben.;;;;;;;;;;;x
 EVTOPTB_WCUND_1301;It is a price I am willing to pay to bask in the power of the Lich King.;;Es ist ein Preis, den ich bereit bin zu zahlen, um mich in der Macht des Lichk�nigs zu sonnen.;;;;;;;;;;;x
 EVTOPTC_WCUND_1301;I shall bind my soul to the Blighted Lands, so I can draw my power from there.;;Ich werde meine Seele an die verw�steten Lande binden, damit ich meine Kraft von dort beziehen kann.;;;;;;;;;;;x
-I_WILL_NOT_BETRAY_RELIGION;I will not betray [Root.TrueReligion.GetHighGodName]!;;Ich werde [Root.TrueReligion.GetHighGodName] nicht verraten!;;;;;;;;;;;x
+I_WILL_NOT_BETRAY_RELIGION;I hope [Root.TrueReligion.GetHighGodName] will forgive me.;;Vergib mir, [Root.TrueReligion.GetHighGodName]!;;;;;;;;;;;x
```
Comment (arithon) [inline localisation/00_zuwc_united_localisation.csv]: Hoffentlich wird mir [Root.TrueReligion.GetHighGodName] vergeben.

Comment (DJScias) [reply localisation/00_zuwc_united_localisation.csv]: What, how did it.. I pushed the changes with the.. German.. What?! I'm confused, I'll fix it but it was fixed locally. >->

Comment (arithon) [reply localisation/00_zuwc_united_localisation.csv]: had that problem once as well, thought that I had pushed something, switched branch, and afterwards it was gone ...

**Comment by DJScias** [comment]
This push should have taken care of it, hopefully. @arithon

**Comment by zumbak04** [comment]
@DJScias , merge?

**Comment by DJScias** [comment]
I'm all fine with it, got nothing more to add/change.

**Pull request merged**